packs/equipment/madcap-top.json

```diff
@@ -9,7 +9,7 @@
e must be the center of the spell's area, if it has an area but no targets. If the spell's range is less than 60 feet, increase the range to 60 feet.</p>\n<p>Any spell DC required is DC 27, and any spell attack roll required is +17. If the top casts a spell on you, you don't get a saving throw or other defense against it.</p>\n<p>The top can't be activated again for [[/br 1d4 #hours]]{1d4 hours}</p><hr /><p>@UUID[Compendium.pf2e.rollable-tables.RollTable.Madcap Top Effect]</p><table border=\"1\" class=\"pf2e\"><thead><tr style=\"text-align:left\"><th>d20</th><th>Madcap Top Effect</th></tr></thead><tbody><tr><td>1</td><td>You spin just like the top. You are @UUID[Compendium.pf2e.conditionitems.Item.Stunned] for 1 round and then @UUID[Compendium.pf2e.conditionitems.Item.Confused] for 1 round.</td></tr><tr><td>2</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Slow] affects the target.</td></tr><tr><td>3</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Shrink] affects you for 1 day.</td></tr><tr><td>4</td><td>A 3rd-rank @UUID[Compendium.pf2e.spells-srd.Item.Illusory Disguise] makes you look like the target.</td></tr><tr><td>5</td><td>A statue of the target, made of chocolate or candy, appears adjacent to you.</td></tr><tr><td>6</td><td>Gravity reverses, sending you and the target 30 feet in the air, immobilized. You both fall at the start of your next turn.</td></tr><tr><td>7</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Mind Reading] affects the target, and the top loudly recites what it discovers.</td></tr><tr><td>8</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Laughing Fit] affects the target.</td></tr><tr><td>9</td><td>A fountain erupts from the top, spraying wine for 10 minutes.</td></tr><tr><td>10</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Translocate] affects you, but you teleport through Hell.</td></tr><tr><td>11</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Invisibility] affects you.</td></tr><tr><td>12</td><td>Four singing skeletons appear to serenade you and your allies for 1 minute, granting a +1 status bonus to attack rolls, Perception checks, saving throws, and skill checks.</td></tr><tr><td>13</td><td>The top flings 1 cp at you.</td></tr><tr><td>14</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Confusion] affects the target.</td></tr><tr><td>15</td><td>You and the target change places; this is a teleportation effect.</td></tr><tr><td>16</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Banishment] affects the target, or paralyze if the target's on its home plane.</td></tr><tr><td>17</td><td>Rainbow dye explodes and covers you for 24 hours.</td></tr><tr><td>18</td><td>The top creates a @UUID[Compendium.pf2e.spells-srd.Item.Toxic Cloud] centered on the target.</td></tr><tr><td>19</td><td>A random unattended object within 60 feet of you sprouts animal legs and follows you, reverting when the top's activation recharges.</td></tr><tr><td>20</td><td>Spin again, and the target also spins as though it had activated the top.</td></tr></tbody></table>"
```

Comment:
```suggestion
            "value": "<p>This top has 20 divisions, painted in a chaotic mess of clashing colors. When spun, the top quickly settles on a side and generates a strange magical effect based on the side that lands face-up.</p>\n<p><strong>Activate—Spin the Top</strong> <span class=\"action-glyph\">2</span> (concentrate, manipulate)</p><hr /><p><strong>Effect</strong> Choose a creature within 60 feet to target and roll a d20 on the table below to determine the top's effect. You make any decisions for a spell cast by the top unless otherwise indicated, except that it must target the creature you chose, or the creature you chose must be the center of the spell's area, if it has an area but no targets. If the spell's range is less than 60 feet, increase the range to 60 feet.</p>\n<p>Any spell DC required is DC 27, and any spell attack roll required is +17. If the top casts a spell on you, you don't get a saving throw or other defense against it.</p>\n<p>The top can't be activated again for [[/br 1d4 #hours]]{1d4 hours}</p><hr /><p>@UUID[Compendium.pf2e.rollable-tables.RollTable.Madcap Top Effect]</p><table class=\"pf2e\"><thead><tr><th>d20</th><th>Madcap Top Effect</th></tr></thead><tbody><tr><td>1</td><td>You spin just like the top. You are @UUID[Compendium.pf2e.conditionitems.Item.Stunned] for 1 round and then @UUID[Compendium.pf2e.conditionitems.Item.Confused] for 1 round.</td></tr><tr><td>2</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Slow] affects the target.</td></tr><tr><td>3</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Shrink] affects you for 1 day.</td></tr><tr><td>4</td><td>A 3rd-rank @UUID[Compendium.pf2e.spells-srd.Item.Illusory Disguise] makes you look like the target.</td></tr><tr><td>5</td><td>A statue of the target, made of chocolate or candy, appears adjacent to you.</td></tr><tr><td>6</td><td>Gravity reverses, sending you and the target 30 feet in the air, immobilized. You both fall at the start of your next turn.</td></tr><tr><td>7</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Mind Reading] affects the target, and the top loudly recites what it discovers.</td></tr><tr><td>8</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Laughing Fit] affects the target.</td></tr><tr><td>9</td><td>A fountain erupts from the top, spraying wine for 10 minutes.</td></tr><tr><td>10</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Translocate] affects you, but you teleport through Hell.</td></tr><tr><td>11</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Invisibility] affects you.</td></tr><tr><td>12</td><td>Four singing skeletons appear to serenade you and your allies for 1 minute, granting a +1 status bonus to attack rolls, Perception checks, saving throws, and skill checks.</td></tr><tr><td>13</td><td>The top flings 1 cp at you.</td></tr><tr><td>14</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Confusion] affects the target.</td></tr><tr><td>15</td><td>You and the target change places; this is a teleportation effect.</td></tr><tr><td>16</td><td>@UUID[Compendium.pf2e.spells-srd.Item.Banishment] affects the target, or paralyze if the target's on its home plane.</td></tr><tr><td>17</td><td>Rainbow dye explodes and covers you for 24 hours.</td></tr><tr><td>18</td><td>The top creates a @UUID[Compendium.pf2e.spells-srd.Item.Toxic Cloud] centered on the target.</td></tr><tr><td>19</td><td>A random unattended object within 60 feet of you sprouts animal legs and follows you, reverting when the top's activation recharges.</td></tr><tr><td>20</td><td>Spin again, and the target also spins as though it had activated the top.</td></tr></tbody></table>"
```